project-base/storefront/gtm/mappers/mapGtmUserEntryInfoFromCurrentCustomer.ts

```diff
@@ -0,0 +1,17 @@
+import { GtmUserEntryInfoType } from 'gtm/types/objects';
+import { CurrentCustomerType } from 'types/customer';
+
+export const mapGtmUserEntryInfoFromCurrentCustomer = ({
+    uuid,
+    email,
+    firstName,
+    lastName,
+    loginInfo: { loginType, externalId },
+}: CurrentCustomerType): GtmUserEntryInfoType => ({
+    id: uuid,
+    email,
+    firstName,
+    lastName,
+    loginType,
+    loginUserId: externalId,
```

Comment:
We would definitely like to avoid this double naming. Your goal should be to either _force_ the BE dev to change the name to match the name needed by GTM, or the other way around. It would only cause confusion later.

You might wonder why I do not mention it regarding ID and UUID. It's because it's a lost battle 😆

Reply:
Ok I will reach out to someone about this issue.

Reply:
Fixed and added comment to the original ticket.